.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 14
+  - 15
```

Comment:
you can use `node` instead of a specific version to use the latest

https://docs.travis-ci.com/user/languages/javascript-with-nodejs

Reply:
Thanks, I did consider doing that, but thought that it would be better to target v15 specifically in case the eventual v16 brings in any breaking changes that would require us to, temporarily, downgrade to 15. I'll update it in the future though!